xfields/beam_elements/beambeam3d.py

```diff
@@ -102,6 +133,7 @@ def __init__(self,
                     phi=None, alpha=None, other_beam_q0=None, particles_per_macroparticle = None,
 
                     slices_other_beam_num_particles=None,
+                    slices_other_beam_num_macroparticles=None,
```

Comment:
If the slice contains 0 particles, the mean and sigma are 0. If the slice contains 1 particle, the sigma is 0. If the sigma is 0 then the cerrf returns nans, so the field will be nan. Instead of the if on the macroparticles I could add an if on sigma > 0, but there has to be an if anyways.

Reply:
I think that would be better

Reply:
see [https://github.com/xsuite/xfields/pull/50#discussion_r1003030751]